tests/plugins/toolsets/datadog/logs/test_utils.py

```diff
@@ -16,7 +17,7 @@
             DatadogConfig(
                 dd_api_key="xyz",
                 dd_app_key="xyz",
-                site_api_url="xyz",
+                site_api_url=AnyUrl("https://example.com"),
```

Comment:
Is the AnyUrl is really needed? I expect it to work with even a string other wise it might be broken when setting it in the config using yaml. Can you please double check to make sure we didn't miss anything?

Reply:
yes it works.